kubectl-fdb/cmd/k8s_client.go

```diff
@@ -393,17 +394,27 @@ func fetchPodsOnNode(kubeClient client.Client, clusterName string, namespace str
 }
 
 // getProcessGroupsByCluster returns a map of processGroupIDs by FDB cluster that match the criteria in the provided
-// processGroupSelectionOptions.
-func getProcessGroupsByCluster(kubeClient client.Client, opts processGroupSelectionOptions) (map[*fdbv1beta2.FoundationDBCluster][]fdbv1beta2.ProcessGroupID, error) {
+// processSelectionOptions.
+func getProcessGroupsByCluster(cmd *cobra.Command, kubeClient client.Client, opts processGroupSelectionOptions) (map[*fdbv1beta2.FoundationDBCluster][]fdbv1beta2.ProcessGroupID, error) {
+	// option compatibility checks
 	if opts.clusterName == "" && opts.clusterLabel == "" {
 		return nil, errors.New("processGroups will not be selected without cluster specification")
 	}
+	if opts.clusterName == "" { // cli has a valid default for clusterLabel
```

Comment:
```suggestion
	if opts.clusterName == "" { // cli has a default for clusterLabel
```

I would remove the `valid` term as it depends on how the cluster where created.